packages/components/src/stories/Introduction.mdx

```diff
@@ -5,10 +5,29 @@ import { Meta } from '@storybook/blocks';
 
 # `@plone/components`
 
-This package contains ReactJS components for Plone 7 and Plone's API-first CMS story.
+This package contains ReactJS basic components for Plone 7 and Plone's frontend modular architecture.
 
 The purpose of this package is to provide an agnostic set of baseline components to build Plone sites upon.
 
+`@plone/components` is based on [React Aria Components](https://react-spectrum.adobe.com/react-aria/components.html), the documentation there applies also to all the components in this package.
+
+## Design decisions and assumptions
+
+This package provides a set of simple, ready to use components with the following features:
+
+- Agnostic (not tied to any known design system)
+- Composable (they are ready to be the building blocks of other complex components)
+- Theme-able (they can be themed, using both the provided basic theme as a baseline and custom CSS properties)
+- Data-driven-less (they are dumb, presentational components)
+- i18n-less (they do not provide i18n machinery or attached to any i18n framework or library)
+- Built on a renowned headless CMS components library: [React Aria Components (RAC)](https://react-spectrum.adobe.com/react-aria/components.html)
+
+Since we are using RAC as the base, the styling of this package components are done via the [RAC styling options](https://react-spectrum.adobe.com/react-aria/styling.html).
```

Comment:
```suggestion
Since we are using RAC as the base, the styling of these package components is done via the [RAC styling options](https://react-spectrum.adobe.com/react-aria/styling.html).
```